spec/support/disable_animation.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Rack
+  # disable CSS3 and jQuery animations in test mode for speed, consistency and avoiding timing issues.
+  # Usage for Rails:
+  # in config/environments/test.rb
+  # config.middleware.use Rack::NoAnimations
+  class NoAnimations
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/4aeae2b3978c089a65c67e956a7e5b3652e0a5e0/spec/support/disable_animation.rb#L8"></span>Assumes too much for instance variable ‘@status’</td>
    </tr>
  </tbody>
</table>